pkg/crc/machine/start.go

```diff
@@ -784,6 +795,25 @@ func addNameServerToInstance(sshRunner *crcssh.Runner, ns string) error {
 	return nil
 }
 
+func enableEmergencyLogin(sshRunner *crcssh.Runner) error {
```

Comment:
I'd start with 
```
if crcos.FileExists(constants.PasswdFilePath) {
    return nil
}
```

and add a `os.Remove(constants.PasswdFilePath)` to `disableEmergencyLogin`.

This would ensure we don't generate a different password each time we do `crc start; crc stop; crc start`. This makes the behaviour consistent with the ssh key, which we create only at VM creation time, not for each start/stop/start cycle.

Reply:
This 'recreate'-behaviour I also did not understand.